autometa/taxonomy/ncbi.py

```diff
@@ -333,22 +311,18 @@ def parse_nodes(self):
         return nodes
 
     def parse_merged(self):
-        """Short summary.
+        """
+        Parses through `merged.dmp` databse and returns a dictionary having old and new taxid
```

Comment:
```suggestion
        Parses through `merged.dmp` database and returns a dictionary having old and new taxid
```

Reply:
Can you refrain from resolving conversations when the comment has not been addressed?